### ISSUE TITLE
User provided network file (Availability + Accessibility)

### ISSUE THREAD

**Issue by YuriGrings** [Spatial-Data-Science-and-GEO-AI-Lab/GreenEx_Py#1]
Ideally, users should be able to provide a network file to the availability and accessibility functions. Only if this file is not provided, the network will be retrieved from OpenStreetMap. 

When user provides a network file, the network should be converted into a graph object to be processed by the availability and accessibility functions. Creating the graph object can be done using e.g. the 'momepy' package. However, the structure of the resulting graph object seems to deviate slightly from the graph objects which are generated by the 'osmnx' package. Consequently, without adjusting for these deviations, osmnx functions cannot be applied to the created graph objects. During the thesis period, I haven't been able to succesfully standardize the graph objects and make a robust workflow for this. This is why in the current version of the package, the network will always be retrieved from OpenStreetMap for the availability and accessibility functions. 

Eventually, isochrones should be computed (availability) and network distances should be calculated (accessibility) for the graph object.
